postgres/templates/limit.maxfiles.plist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>  
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"  
+        "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">  
+  <dict>
+    <key>Label</key>
+    <string>limit.maxfiles</string>
+    <key>ProgramArguments</key>
+    <array>
+      <string>launchctl</string>
```

Comment:
Probably best to do the full path here. `/bin/launchctl`.

Reply:
Thank you for reviewing @weswhet,  Good spot, I fixed that issue!!